_posts/pl/2022-10-11-healthcheck.md

```diff
@@ -0,0 +1,85 @@
+---
+layout:    post
+title:     "Czy wiesz co to healthcheck i jak z niego korzystać?"
+date:      2022-10-11 06:00:00 +0100
+published: true
+didyouknow: true
+lang: pl
+author:    kczarnecki
+image:     /assets/img/posts/2022-10-11-healthcheck/healthcheck.jpg
+tags:
+
+- liveness probe
+- heathcheck
+- kubernetes
+
+---
+
+# Czy wiesz co to healthcheck i jak z niego korzystać?
```

Comment:
na stronie powinien być tylko jeden nagłówek pierwszego stopnia i jest to tytuł dodawany przez jekylla. Trzeba podwyższyć stopień wszystkich nagłówków w tym artykule